packages/sanity/src/core/studio/workspace.tsx

```diff
@@ -1,16 +1,19 @@
-import {type ReactChild, useContext} from 'react'
+import {useContext} from 'react'
 import {WorkspaceContext} from 'sanity/_singletons'
 
 import {type Workspace} from '../config'
 
 /** @internal */
 export interface WorkspaceProviderProps {
   workspace: Workspace
-  children?: ReactChild
+  children: React.ReactNode
```

Comment:
It never made sense for this provider to have children be optional. And `ReactChild` is deprecated, the official `React.PropsWithChildren` uses `ReactNode` and so should we :)